### PR TITLE
To be merged Nov 17th - release(Jakarta): release Jakarta app services

### PR DESCRIPTION
**Pull request by bill-mahoney**
Signed-off-by: Bill Mahoney <bill.mahoney@intel.com>

### CODE REVIEW

File: release/app-service-configurable.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: 'app-service-configurable'
+version: '2.1.0'
+releaseName: 'jakarta'
+releaseStream: 'main'
+lts: true
+repo: 'https://github.com/edgexfoundry/app-service-configurable.git'
+commitId: '4ec47cdd8cea7a0176fb01ed614db2877bbe7fef'
```
Comment (lenny-intel) [inline release/app-service-configurable.yaml:8]:
```suggestion
commitId: 'bfff9069ec602defc1c67fa58b1ab94b3f5a4b73'
```
That is the final commit for release.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by ernestojeda — APPROVED** [review]
ready, set, go!

**Pull request merged**